src/Dialect/ONNX/ONNXShapeHelper.hpp

```diff
@@ -45,12 +47,24 @@ struct ONNXOpShapeHelper {
     llvm_unreachable("implement in child structs");
   }
 
+  // Return output dims for the N-th output.
+  DimsExpr &getDimsForOutput(int n) {
```

Comment:
Nit: Get/Set goes typically in pairs, and here we have a get that is used for both as you return the reference. So maybe just name it 'dimsForOutput(int n)'. Note that I believe SmallVector already assert for out of bound accesses.

Reply:
Done.